sql/calendar.sql

```diff
@@ -0,0 +1,25 @@
+DROP TABLE IF EXISTS calendar_logs;
+DROP TABLE IF EXISTS calendar_events;
+CREATE TABLE calendar_logs(
```

Comment:
This table looks unused. Is it needed?

Reply:
It's logging everyone who requests to share a calendar with their gmail

Reply:
Ah, sorry, the diff for the `helpers.py` file was hidden when I Ctrl-Fed for it.